org.cispa.Scyther.json

```diff
@@ -0,0 +1,139 @@
+{
+   "app-id":"org.cispa.Scyther",
```

Comment:
Does this domain actually exist? Seems to be a generic landing page by domain squatters.